app/main.py

```diff
@@ -1,8 +1,51 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int
+                 ) -> None:
```

Comment:
In case you do not have any arguments on the same line as the closing parenthesis, your opening parenthesis should not have any arguments on the same line as well. Reformat your code to encounter that and make the `__init__` style the same as for the `Car` class.

Reply:
@Dimosphen1 I got, thanks for explanation)